enterprise/signals.py

```diff
@@ -79,3 +86,16 @@ def default_content_filter(sender, instance, **kwargs):     # pylint: disable=un
     if kwargs['created'] and not instance.content_filter:
         instance.content_filter = get_default_catalog_content_filter()
         instance.save()
+
+
+@receiver(post_save, sender=EnterpriseCustomerUser)
+def assign_enterprise_learner_role(sender, instance, **kwargs):     # pylint: disable=unused-argument
```

Comment:
i think we will need a similar mechanism to delete the role assignment when enterprise customer user records are deleted.